setup.py

```diff
@@ -118,3 +118,8 @@
       entry_points=entry_points,
       **package_info
 )
+
+#Compiling the resource file for the gui
+import subprocess
```

Comment:
I think we should use the structure that `glue-viz` uses - that is a proper use of the setup structure and ensures that things are working.